packages/client/components/EditorLinkChanger/EditorLinkChangerDraftjs.tsx

```diff
@@ -51,21 +51,19 @@ const EditorLinkChangerDraftjs = (props: Props) => {
   } = props
   useTaskChild('editor-link-changer')
   const handleSubmit = ({text, href}: {text: string; href: string}) => {
-    const keepSelection = true
     // don't include leading and trailing whitespace in the link text
     const nonWhitespaceFromStart = text.search(/\S/)
     const nonWhitespaceFromEnd = text.search(/\s*$/)
     const startStr = nonWhitespaceFromStart === -1 ? '' : text.slice(0, nonWhitespaceFromStart)
     const endStr = nonWhitespaceFromEnd === -1 ? '' : text.slice(nonWhitespaceFromEnd)
-    const trimmedText = text.trim() ? text.trim() : text
+    const hasTextTitle = text.trim()
```

Comment:
+1 Name is not ideal as it implies to be a boolean.
It would be nicer to do
```ts
const hasTextTitle = !!text.trim()
// ...
const trimmedText = hasTextTitle ? text.trim() : text
```
This introduces back a bit of redundant code, but improves readability. My fault for leaving the comment in there with `||` after adding the other one about the variable.
```

Reply:
Ok